rulesets/resources/ES_DENHruleset.rb

```diff
@@ -492,11 +492,15 @@ def self.set_enclosure_floors_reference(orig_bldg, new_bldg)
       next unless orig_floor.is_floor
 
       subtype = orig_floor.floor_type == HPXML::FloorTypeConcrete ? 'mass' : 'wood'
-      floor_ufactor = lookup_reference_value('floors_over_uncond_spc_ufactor', subtype)
-      floor_ufactor = lookup_reference_value('floors_over_uncond_spc_ufactor') if floor_ufactor.nil?
+      floor_ufactor = lookup_reference_value('floors_ufactor', subtype)
+      floor_ufactor = lookup_reference_value('floors_ufactor') if floor_ufactor.nil?
 
-      if orig_floor.is_thermal_boundary && (not [HPXML::LocationOtherHousingUnit, HPXML::LocationOtherHeatedSpace].include?(orig_floor.exterior_adjacent_to))
-        # This is meant to apply to floors over unconditioned spaces, non-freezing spaces, multifamily buffer boundaries, or the outdoor environment
+      subtype = orig_floor.exterior_adjacent_to
+      floor_insulated = lookup_reference_value('floors_insulated', subtype)
+      floor_insulated = true if floor_insulated.nil?
+
+      if orig_floor.is_thermal_boundary && floor_insulated
+        # This is meant to apply to floors over unconditioned spaces, non-freezing spaces, unrated heated spaces, multifamily buffer boundaries, or the outdoor environment
```

Comment:
Also, this comment no longer makes sense given that the types of floors that this applies to is now conditional (dependent on program/version).